sdk/spring-sdk/src/it/java/com/example/wiring/SpringSdkWiringIntegrationTest.java

```diff
@@ -107,13 +107,18 @@ public void verifyCounterEventSourceSubscription() {
         .bodyToMono(Integer.class)
         .block(timeout);
 
-    Integer counterAfterAction = 0;
     // THEN IncreaseAction receives the event 42 and increases the counter 1 more
-    while (counterAfterAction != 43) {
-      counterAfterAction =
-          webClient.get().uri("/counter/hello").retrieve().bodyToMono(Integer.class).block(timeout);
-    }
-    Assertions.assertTrue(true);
+    await()
+        .atMost(20, java.util.concurrent.TimeUnit.SECONDS)
```

Comment:
we recently learned that we should add a `.ignoreExceptions` before the `.atMost`. 

The reason is that if the call fails if an exception, the test fails. It keeps trying until it gets 43, but fails on exception. 

That might not be so important for this case, as we know that the call will succeed each time, but not always returning 43. Still, good to call `.ignoreExceptions` for good measure.

Reply:
added